src/float.js

```diff
@@ -2,6 +2,8 @@
  * @namespace Sk.builtin
  */
 
+const biginteger = require("big-integer");
```

Comment:
Lets call this BigInt so we only have to remove this
line in the future.

Reply:
I don't think that will work the way you are proposing.  The "biginteger" object doesn't have the same interface as "BigInt".  I feel that calling it BigInt would be problematic because the rest of the code would look broken, as it would be calling methods that native BigInts don't have...  And then when we do go to native BigInts, we would still have to change all the code relating to big integers.

Reply:
I was under the impression it was a drop in replacement

Reply:
It's not possible to build a polyfill that is a drop in replacement.  With native BigInts, you add/subtract/etc. with the "+", "-", etc. operators.  They can't be overloaded.  So, there are two realistic options for dealing with browsers that don't support BigInt:

1) Using a library like this that has a function-based API for big int operations, where they internally use native BigInts if/when they can.  The code looks different than it would if it solely used native BigInts, but the performance should be close on systems that support BigInt.

2) Write the source code using native BigInts and use a compiler to convert the appropriate operations to library calls.  This is known to not work in a variety of cases.  The only advantage being that when you switch to all native BigInts, the code doesn't need to change.  But, the correctness problems in the interim would seem to be enough of a deal breaker.